packages/utilities/src/customizations/Customizations.ts

```diff
@@ -71,6 +76,18 @@ export class Customizations {
     return settings;
   }
 
+  /** Used to run some code that sets Customizations without triggering an update.
+   * Useful for applying Customizations that don't affect anything currently rendered.
+   */
+  public static suppressUpdates(code: () => void): void {
+    Customizations._suppressUpdates = true;
+    try {
+      code();
+    } finally {
+      Customizations._suppressUpdates = false;
```

Comment:
Hmm. it will suppress updates, but then, when will the updates raise changes?

Reply:
(I expected that we'd track something like a pending change required, and then at the end if that's true, raise change.

Reply:
Suggested tweak:

```tsx
public static applyBatchedUpdates(cb: () => void, supressChanges: false)
```